cmd/nginx-ingress/main.go

```diff
@@ -445,6 +451,16 @@ func getSocketClient(sockPath string) *http.Client {
 	}
 }
 
+// validateLockName validates the name of a resource
+// It returns an error if the resource name is invalid
+func validateLockName(lock string) error {
+	lst := validation.IsDNS1123Subdomain(lock)
```

Comment:
Lets use a more appropriate variable name. 
```suggestion
	allErrs := validation.IsDNS1123Subdomain(lock)
```